.github/workflows/tests.yaml

```diff
@@ -4,18 +4,18 @@ on:
   pull_request:
     types: [opened, synchronize]
     paths:
-      - 'api/**'
-      - 'managers/**'
-      - 'mappings/**'
-      - 'model/**'
-      - 'processes/**'
-      - 'services/**'
-      - 'tests/**'
-      - 'utils/**'
-      - 'dev-requirements.txt'
-      - 'requirements.txt'
-      - 'main.py'
-      - 'Makefile' 
+      - "api/**"
+      - "managers/**"
+      - "mappings/**"
+      - "model/**"
+      - "processes/**"
+      - "services/**"
+      - "tests/**"
+      - "utils/**"
+      - "dev-requirements.txt"
+      - "requirements.txt"
+      - "main.py"
+      - "Makefile"
```

Comment:
Do you have a formatter setup for yaml?

If so, recommend turning it off in this repo. These are fine but it creates unnecessary changes to the file.